openmetadata-ui/src/main/resources/ui/src/pages/teams/AddUsersModal.tsx

```diff
@@ -60,9 +60,11 @@ const AddUsersModal = ({
       })
       .map((user, index) => {
         const User = {
-          description: user.displayName || user.description || '',
-          name: user.name || '',
+          displayName: user.displayName || user.name || '',
```

Comment:
Can we define a static utility to give a name string from EntityReference and use it everywhere instead of `user.displayName || user.name || ''`?

Reply:
@vivekratnavel , Yeah, I will make the changes.